website/docs/ref/core.md

```diff
@@ -153,6 +153,28 @@ i18n._("My name is {name}", { name: "Tom" })
 i18n._("msg.id", { name: "Tom" }, { message: "My name is {name}" })
 ```
 
+### `i18n.t(messageDescriptor) (experimental)` {#i18n.t}
+
+A small wrapper on the core translation meant for NodeJS/JS usage without macros. It uses the core `_` method, but currently only accepts message descriptor. This API is prone to breaking changes
+
+`messageDescriptor` is an object of message parameters.
```

Comment:
@andrii-bodnar could you help @j4hr3n here to write more descriptive doc for the method? I think it should be stated from the doc that the main difference here is how this method is treated by extractor. (not the signature of the method, how some may understand from this doc)

And probably we also could state that if someone wants to use lingui without macro it's better to start from this method.

Reply:
I'm going to release it within the `v4.0.0-next.5` and then check the possibility of docs improvement in a separate PR